ipi-java-eval-master/src/main/java/com/ipiecoles/java/javaeval/MyRunner.java

```diff
@@ -0,0 +1,266 @@
+		            }
+	        	}
+	        }
+	        boolean continueEts = true;
+	        while(continueEts){
+		        Tools.affichage4(); // Page principale de gestion de l'entreprise
+		        choice = sc.nextLine();
+		        
+		        //Ajouter un employé à l'entreprise
+		        if(choice.equals("1")) {
+		        	 Tools.affichage5();//Page d'ajout d'un employé
+		
+		             System.out.print("Nom : ");
+		         	 String nom = sc.nextLine();
+		
+		             System.out.print("Prénom : ");
+		         	 String prenom = sc.nextLine();
+		
+		             System.out.print("Date d'embauche : (jj-mm-aa)");
+		         	 String date = sc.nextLine();
+		         	 LocalDate datemb = new LocalDate(date);
+		         	 
+		
+		             System.out.print("Matricule : ");
+		         	 String mat = sc.nextLine();
+		         	 
+		         	 System.out.print("Salaire : ");
+		         	 String salaire = sc.nextLine();
+		         	         	
+		             print("Type d'employe ? (1 : Cadre, 2 : Manager, 3 : Commercial, 4 : Technicien) ");
+		             choice = sc.nextLine();
+		             if(choice .equals("1")) {
+		            	 Cadre emp = null;
+		            	 while(emp == null) {
+			            	 print("Indiquez son coefficient : ");
+			            	 try {
+			            		 	emp = new Cadre(nom,prenom,mat,datemb,Double.valueOf(salaire),Integer.valueOf(sc.nextLine()));
+			            	 }catch(CadreException e) {}
+		            	
+			             }	 
+			            	 etsserv.addEmployeToEntreprise(emp, ets);
+		             }
+		             else if (choice.equals("2")) {
+		            	 Manager emp = new Manager(nom,prenom,mat,datemb,Double.valueOf(salaire));
+		            	 etsserv.addEmployeToEntreprise(emp, ets);
+		             }
+		             else if(choice.equals("3")) {
+		            	 Commercial emp = new Commercial(nom,prenom,mat,datemb,Double.valueOf(salaire),0.0);
+		            	 etsserv.addEmployeToEntreprise(emp, ets);
+		             }
+		             else if(choice.equals("4")) {
+			         	 Technicien emp = null;
+			         	 while(emp==null) {
+			            	 print("Indiquez son grade : ");
+			            	 Integer grade = Integer.valueOf(sc.nextLine());
+			            	 try {
+			            		 	emp = new Technicien(nom,prenom,mat,datemb,Double.valueOf(salaire),grade);
+			            	 }catch(TechnicienException e) {}
+			         	 }
+		                 etsserv.addEmployeToEntreprise(emp, ets);
+		             }
+		        }
+		        
+		        //Lister les employés de l'entreprise
+		        else if (choice.equals("2")) {
+		        	List<Employe>emp = (etsserv.getEmployes(ets));
+		        	for(int i=0; i<3;i++)print(" ");//De l'espace
+		        	
+		        	print("Les employés de votre entreprise sont : ");
+		        	print(" ");
+		        	for(Iterator<Employe> it=emp.iterator(); it.hasNext();) 
```

Comment:
Eviter les for sans accolades, les mettre systématiquement même s'il n'y a qu'une instruction